### PR TITLE
Hotfix/force cache invalidation on page refresh

### PR DESCRIPTION
**Pull request by mlt47**

### CODE REVIEW

File: src/components/semantic/presenters/FigurePresenter.tsx [hunk 1]
```diff
@@ -99,12 +98,12 @@ export function FigurePresenter(props: PresenterProps<Figure>) {
...
             if (src) {
-                setReplacedFile(src === docRef.current.src);
-                setLastUpdated(Date.now());
-                update({
-                    ...docRef.current,
-                    src,
-                });
+                if (src === docRef.current.src) {
+                    setReplacedFile(true);
+                    setTimeout(() => { setReplacedFile(false) }, 3000);
```
Comment (mlt47) [inline src/components/semantic/presenters/FigurePresenter.tsx:103]: This is an addition to make the replaced image banner disappear after 3s. It is possible that that might cause confusion

Comment (chrisjpurdy) [reply src/components/semantic/presenters/FigurePresenter.tsx:103]: I think the banner should stay IMO. If content say that it's in the way at some point, we could put a cross in the top corner, allowing them to dismiss it themselves.

**Review by chrisjpurdy — APPROVED** [review]
All looks good to me! Will leave it up to you whether to keep the "image replaced" banner timeout, but personally I don't really like that approach.

File: src/components/semantic/presenters/FigurePresenter.tsx (hunk 1 above)
Comment (chrisjpurdy) [inline src/components/semantic/presenters/FigurePresenter.tsx:103]:
```suggestion
```

**Pull request merged**